papaparse.js

```diff
@@ -415,6 +415,24 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
```

Comment:
It normalizes the different types (functions, objects, or boolean) a user can configure `dynamicTyping` with.